solr/core/src/java/org/apache/solr/handler/admin/SystemInfoHandler.java

```diff
@@ -149,7 +145,7 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
     rsp.add( "security", getSecurityInfo(req) );
     rsp.add( "system", getSystemInfo() );
     if (solrCloudMode) {
-      rsp.add("node", getCoreContainer(req, core).getZkController().getNodeName());
+      rsp.add(PARAM_NODE, getCoreContainer(req, core).getZkController().getNodeName());
```

Comment:
PARAM_NODE is called that because it's a _parameter_.  It's use here in a response structure is dubious because it's not a parameter.  Other keys of the response above just use static strings.

Reply:
PARAM_NODE is initialized but not used in this class. Then I thought it can be used here;

Reply:
Okay but that doesn't mean it should be used in a non-param way.

Let's not touch SystemInfoHandler or it's registration in this issue/PR.